src/components/dispenser-item.tsx

```diff
@@ -36,20 +37,28 @@ export const DispenserItem = React.forwardRef(function(
       case RomiCore.DispenserState.OFFLINE:
         return `${classes.dispenserLabel} ${dispenserModeLabelClasses.offline}`;
       default:
-        return '';
+        return `${classes.dispenserLabel} ${dispenserModeLabelClasses.unknown}`;
     }
   }
 
-  function dispenserRequestQueueId(): React.ReactElement{
+  function dispenserRequestQueueId(): JSX.Element {
```

Comment:
Just checking, Is this JSX.Element or React.ReactElement?

Reply:
Should be React.Element. Just changed it